patterns/banner-with-description-and-images-grid.php

```diff
@@ -11,11 +11,11 @@
  */
 
 ?>
-<!-- wp:group {"align":"full","layout":{"type":"constrained"}} -->
-<div class="wp-block-group alignfull">
-	<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"},"blockGap":"var:preset|spacing|60"}},"layout":{"type":"grid","minimumColumnWidth":"26rem"}} -->
-	<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--60);padding-bottom:var(--wp--preset--spacing--60)">
-		<!-- wp:group {"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"space-between","justifyContent":"stretch"}} -->
+<!-- wp:group {"align":"full","className":"is-style-section-2","layout":{"type":"constrained"}} -->
```

Comment:
If we follow [the design](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1816&t=I9366XAWNgeutunw-1), we shouldn't include the section style for this one.

Reply:
Resolved, thanks! https://github.com/WordPress/twentytwentyfive/pull/397/commits/5dacf014ae1f31a107ac73d496b749a1bf93b203